inst/include/smc-exception.h

```diff
@@ -38,7 +38,14 @@
 #define SMCX_MISSING_HISTORY 0x0010
 ///Exception thrown if an attempt is made to instantiate a class of which a single instance is permitted more than once.
 #define SMCX_MULTIPLE_INSTANTIATION 0x1000
-
+///Exception thrown if an attempt is made to instantiate a class of which a single instance is permitted more than once.
+#define SMCX_MULTIPLE_INSTANTIATION 0x1000
+//Exception thrown if an attempt is made to instantiate a class of which a single instance is permitted more than once.
+#define SMCX_MULTIPLE_INSTANTIATION 0x1000
+//Exception thrown if an attempt is made to use facilities of the base sampler class related to MCMC moves.
+#define CSMCX_USING_MCMC 0x1000
```

Comment:
It would be good to define unique codes for each type of exception so that catching routines can differentiate between difference problems.

Reply:
Done.